packages/lodestar/src/chain/forkChoice/arrayDag/lmdGhost.ts

```diff
@@ -341,7 +342,7 @@ export class ArrayDagLMDGHOST implements ILMDGHOST {
   }
 
   public headStateRoot(): Uint8Array {
-    assert(Boolean(this.justified));
+    assert.true(Boolean(this.justified), "Justified checkpoint does not exist");
```

Comment:
Maybe we should standardize between capitalized and non-capitalized errors. No preference for either, just consistency